src/server.rs

```diff
@@ -226,7 +226,7 @@ fn collect_changes(
             }
 
             for component_id in archetype.components() {
-                let Some(replication_id) = replication_rules.get_id(component_id) else {
+                let Some(&replication_id) = replication_rules.get_ids().get(&component_id) else {
                     continue;
```

Comment:
Will this function just be unlined by the compiler?

Reply:
Maybe just slap `inline`?
Because creating intermediate variable just for getter makes the code noisy.

Reply:
I would prefer to just use `inline` here. Because an explicit method will have a long name, a short one like `get_id` will be confusing. So I would keep it simple.
There are other similar places in the this loop, like `world.storages()` and `archetype_entity.entity()` and they also mark them as inline in Bevy.